src/Compilers/CSharp/Portable/Lowering/SpillSequenceSpiller.cs

```diff
@@ -1132,7 +1132,8 @@ public override BoundNode VisitConditionalOperator(BoundConditionalOperator node
             }
             else
             {
-                var tmp = _F.SynthesizedLocal(node.Type, kind: SynthesizedLocalKind.Spill, syntax: _F.Syntax);
+                var tmp = _F.SynthesizedLocal(node.Type, kind: SynthesizedLocalKind.Spill, syntax: _F.Syntax,
+                    refKind: node.GetRefKind());
```

Comment:
>refKind: node.GetRefKind() [](http://example.com/codeflow?start=20&length=26)

I assume there are scenarios when this temp cannot be preserved across await operations that follow ```BoundConditionalOperator``` in the same expression. We should have a test for a scenario like that.